crates/autopilot/src/driver_model.rs

```diff
@@ -137,8 +137,8 @@ pub mod solve {
     #[derive(Clone, Debug, Default, Deserialize)]
     #[serde(rename_all = "camelCase", deny_unknown_fields)]
     pub struct Solution {
-        /// Unique ID of the solution, used to identify it in subsequent
-        /// requests (reveal, settle).
+        /// Unique ID of the solution (per driver competition), used to identify
+        /// it in subsequent requests (reveal, settle).
```

Comment:
Should we maybe make this unique per driver to prevent some unintended bugs?

Reply:
(fine as is, just food for thought).

Reply:
I think we can do this when we implement sending multiple solutions within `driver`. 
I captured this concern in the issue: https://github.com/cowprotocol/services/issues/1920